File: Codewars/8kyu/8kyu_is_n_divisible_by_x_and_y/src/main.py
def is_divisible(n, x, y):
    return not n % x and not n % y

print(is_divisible(3, 3, 4) == False)
print(is_divisible(12, 3, 4) == True)
print(is_divisible(8, 3, 4) == False)
print(is_divisible(48, 3, 4) == True)

File: Codewars/8kyu/8kyu_is_n_divisible_by_x_and_y/src/main.js
function mysolution(n, x, y) {
    return !(n % x) && !(n % y)
}

function othersolution(n, x, y) {
    return !(n%x||n%y)
}

isDivisible = othersolution

console.log(isDivisible(3, 3, 4) === false);
console.log(isDivisible(12, 3, 4) === true);
console.log(isDivisible(8, 3, 4) === false);
console.log(isDivisible(48, 3, 4) === true);